### ISSUE TITLE
Can I use this tool to normalize audio files?

### ISSUE THREAD

**Issue by davidospina** [PHP-FFMpeg/PHP-FFMpeg#328]
| Q              | A
| -------------- | ---
| Bug?           | no
| New Feature?   | maybe
| Version Used   | I don't know how to find out
| FFmpeg Version | git-2017-01-22-f1214ad
| OS             | Ubuntu 14.04.5 x64

Hi. I've using this tool for file conversions but I wonder if it can be used to normalize audio files to a specific volume level like ## db.

BTW Thanks for your amazing job.

**Comment by jens1o** [comment]
no, it's not possible to do that, yet. But it's a great idea. I'm making a pr.

**Comment by MrMuminov** [comment]
> no, it's not possible to do that, yet. But it's a great idea. I'm making a pr.

Is there a result?

**Comment by jens1o** [comment]
As far as I can tell, there is no result. :/

**Comment by websealevel** [comment]
@jens1o and now ? : )

**Comment by jens1o** [comment]
still not, maybe you want to work on it and file a pull request? :)

**Comment by websealevel** [comment]
Yes, I would like to contribute back for sure : ) 
I will see if I manage to get to know the codebase enough and the process of normazling to file a PR one day
Have a nice day
